RadarSDK.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name                  = 'RadarSDK'
-  s.version               = '3.3.0'
+  s.version               = '3.3.1-beta.1'
```

Comment:
Should this be the same number as it is in Example/Example.xcodeproj/project.pbxproj?

Reply:
Good question, and the answer is no. :) TestFlight & the App Store only support version numbers that consist of `.`-separated integers, and they get the version number from the `.pbxproj`'s `MARKETING_VERSION` value. But CocoaPods (and everything else) _does_ support version numbers like this. The compromise is to use this style in the `.podspec` and in `RadarUtils.m`, and the other style in the `.pbxproj`.